bf2fc6cc711aee1a0c2a/kas-fleet-manager
# Mgdstrm 8476

## Description

Returns a new field `maturity_status` from the supported instance types endpoint.

## Verification Steps

1. Compile and run the fleet manager locally: `make binary && ./kas-fleet-manager serve`
2. Run the OpenAPI server: `make run/docs`
3. Query the `/api/kafkas_mgmt/v1/instance_types/{cloud_provider}/{cloud_region}` endpoint using `aws` and `us-east-1`.
4. Make sure that all sizes have a field `maturity_status`, but only `standard.x1` is `stable`. The others should be `preview`.

## Checklist (Definition of Done)
